### PR TITLE
Consulting page - add UI UX page

### LINKED ISSUE #363: 📄 Consulting Page - UI/UX Design Showcase
https://www.ssw.com.au/ssw/Consulting/UI-UX-Design.aspx

Don't migrate the "load more" button for now

### PR DESCRIPTION
**Pull request by amankumarrr**
closes #363 

URL: `/consulting/ui-ux-design`

### CODE REVIEW

**Comment by wicksipedia** [comment]
Breadcrumb has funny casing

64A3A280-6134-4C6F-9EB2-566B2C79D4B5

**Comment by wicksipedia** [comment]
Some of the tags aren’t visible on mobile. Should they wrap?

2CA15264-6515-415B-9206-613A5455D248

File: components/blocks/internalCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	);
+};
+
+const renderBody = ({ header, paragraph, website, technologies }) => {
+	return (
+		<div key={header} className={header ? "" : "hidden"}>
+			<div className="mt-2 flex justify-between text-left font-semibold text-sswRed prose-p:py-0">
+				<h4>{header}</h4>
+				<span className={website ? "" : "hidden"}>
+					<a href={website} target="_blank">
```
[the remark this answers is not in the document]
Comment (amankumarrr) [reply components/blocks/internalCarousel.tsx]: Fixed

File: content/consulting/software-audit.mdx [hunk 2]
```diff
@@ -1,20 +1,20 @@
 ---
 seo:
-  title: " Software Auditing"
+  title: ' Software Auditing'
```
Comment (wicksipedia) [inline content/consulting/software-audit.mdx]: Why is there a space at the beginning?

Comment (amankumarrr) [reply content/consulting/software-audit.mdx]: I think I accidentally added it, just removed it.

**Review by wicksipedia — APPROVED** [review]
lgtm

**Pull request merged**